categories/data.json

```diff
@@ -31,8 +31,8 @@
 		"name": "Comparison",
 		"color": "cyan"
 	},
-	"graphql": {
-		"name": "GraphQL",
-		"color": "brown"
+	"api-security": {
```

Comment:
Is there any reason why you removed the "GraphQL" category?

Reply:
It happened by mistake. Just fixed it.